app/src/main/java/com/github/se/wanderpals/ui/screens/dashboard/DashboardStopWidget.kt

```diff
@@ -138,11 +138,11 @@ fun DashboardStopWidget(viewModel: DashboardViewModel, onClick: () -> Unit = {})
                       } else {
                         Column {
                           StopWidgetItem(stop = sortedStops[0])
-                          HorizontalDivider(
-                              color = MaterialTheme.colorScheme.surfaceVariant,
-                              thickness = 1.dp,
-                              modifier = Modifier.padding(horizontal = 8.dp))
                           if (sortedStops.size > 1) {
+                            HorizontalDivider(
```

Comment:
Can create a separate function for divider component which will receive all the details in its parameters, and it can be called anywhere in the code.

Reply:
Thank you for your interest in our project. However for the moment this repository is actively being graded for our course.